assets/src/scss/pages/post/_article-content.scss

```diff
@@ -221,20 +216,6 @@
     }
   }
 
-  .more-link {
```

Comment:
The only place in our Github repos where this is still added to the template is on the Lux's site's contact form, where custom CSS overrides all styles anyway.
https://github.com/search?q=org%3Agreenpeace+more-link&type=code

Reply:
The only difference is the large style. This could be solved by giving the [single usage](https://github.com/greenpeace/planet4-child-theme-luxembourg/blob/caf1f950debf95cb8b3a1d201140a3ebc3ccbca0/templates/contact.twig#L144-L146) the `.btn-large` class.